### PR TITLE
Cleanup envelope status and event types

### PR DESCRIPTION
**Pull request by aliveni**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/BPS-517

### Change description ###
- Remove unused envelope status `PROCESSED` 
and event types `DOC_PROCESSED` and `BLOB_DELETE_FAILURE`. 

- Raised https://tools.hmcts.net/jira/browse/RDO-8132 to check if there are any envelopes exist with `PROCESSED` status.

- `BLOB_DELETE_FAILURE` event type database entries were removed in https://github.com/hmcts/bulk-scan-processor/pull/1230

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

**Comment by aliveni** [comment]
No envelopes exist with status `PROCESSED`. so it can be safely deleted. 
see comments in https://tools.hmcts.net/jira/browse/RDO-8132

File: src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/model/common/Event.java [hunk 1]
```diff
@@ -10,9 +10,7 @@
     DOC_SIGNATURE_FAILURE, // Signature verification failure while processing zip file
     DOC_UPLOADED,
     DOC_UPLOAD_FAILURE,
-    DOC_PROCESSED, // when blob is successfully processed after storing all docs in DM. TODO remove when DB is cleaned
     DOC_CONSUMED, // client service handled the documents
-    BLOB_DELETE_FAILURE, // when blob is not successfully deleted after processing is complete
```
Comment (rkondratowicz) [inline src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/model/common/Event.java:15]: I think there are events of this type in DB. If we ever try to read them we'll get an error.

Comment (aliveni) [reply src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/model/common/Event.java:15]: They were deleted in #1230. should I check the database again?

Comment (lgonczar) [reply src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/model/common/Event.java:15]: Worth checking the count of each type of event. It looks like we could delete quite a few of them.

Comment (rkondratowicz) [reply src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/model/common/Event.java:15]: actually, those were removed in a migration, oops

**Comment by rkondratowicz** [comment]
has it also been verified that those to event types deleted in this PR are no longer in the database?

**Comment by aliveni** [comment]
> has it also been verified that those to event types deleted in this PR are no longer in the database?

I've raised a ticket to verify the events types - https://tools.hmcts.net/jira/browse/RDO-8141 
There are database entries for `DOC_PROCESSED` event but no entries exist for `BLOB_DELETE_FAILURE` event. 
`DOC_PROCESSED` events will be deleted after https://github.com/hmcts/bulk-scan-processor/pull/1356. 
This PR can be merged after https://github.com/hmcts/bulk-scan-processor/pull/1356

**Pull request merged**